ios/FluentUI/PreviewCard/PreviewCard.swift

```diff
@@ -8,16 +8,18 @@ import SwiftUI
 
 /// UIKit wrapper that exposes the SwiftUI PreviewCard implementation.
 @objc open class MSFPreviewCard: NSObject, FluentUIWindowProvider {
-    /// The UIView representing the Activity Indicator.
+    /// The UIView representing the PreviewCard.
     @objc open var view: UIView {
         return hostingController.view
     }
     /// Creates a new MSFPreviewCard instance.
     /// - Parameters:
     ///   - theme: The FluentUIStyle instance representing the theme to be overriden for this PreviewCard.
-    @objc public init(theme: FluentUIStyle?, isElevated: Bool = false) {
+    ///   - isElevated: Boolean determining if PreviewCard is flat or elevated.`
+    ///   - placeholderImage: The UIImage assigning an image over PreviewCard
```

Comment:
I may just not understand the assignment, but is it really a "placeholder" image? To me "placeholder" implies temporary or not the real image. Is that the intent?

Reply:
Makes sense from the demo app's perspective, but the `PreviewCard` itself doesn't know the difference. So with that in mind it may make more sense to just call it `image` and then it can be called a placeholder in the demo app.